ncl/ncl_entries/dewtemp.ipynb

```diff
@@ -0,0 +1,122 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# dewtemp_trh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "NCL's [`dewtemp_trh`](https://www.ncl.ucar.edu/Document/Functions/Built-in/dewtemp_trh.shtml) calculates the dew point temperature given temperature and relative humidity using the equations from John Dutton's _\"Ceaseless Wind\"_ (pg. 273-274) and returns a temperature in Kelvin.\n",
+    "\n",
+    ":::{important}\n",
```

Comment:
This isn't rendering properly in the preview.

I'm not quite sure what the problem is offhand.

Reply:
I'm not sure, I changed it back to the div structure since it's been more consistent in rendering